src/pages/CatShow.test.js

```diff
@@ -0,0 +1,9 @@
+import { render } from "@testing-library/react"
+import CatShow from "./CatShow"
+
+describe("<CatShow />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(<CatShow />, div)
+  })
+})
```

Comment:
This doesn't test anything in particular about the catshow page. Set up a test that will sheck if given a cat it will show up.